src/sentry/notifications/platform/provider.py

```diff
@@ -25,12 +27,30 @@ class NotificationProvider[RenderableT](Protocol):
     """
 
     key: NotificationProviderKey
-    default_renderer: type["NotificationRenderer[RenderableT]"]
-    target_class: type["NotificationTarget"]
+    default_renderer: NotificationRenderer[RenderableT]
+    target_class: type[NotificationTarget]
```

Comment:
Added future annotations to alleviate this, though I'm not sure we needed quoted forward declarations for these before?